org-formation/720-client-vpn/_tasks.yaml

```diff
@@ -97,6 +97,12 @@ Vpn:
         AccessGroups:
           - "aws-admins"
           - "bridge-developers"
+      # org-sagebase-synapsedw
+      synapsedw-vpc:
+        CIDR: "10.12.0.0/16"
+        AccessGroups:
+          - "aws-admins"
+          - "aws-synapse-admins"
```

Comment:
We'll need more than admins, users of the datawarehouse snapshots will need access (they run a job to provision an RDS instance and then connect to the instance to run queries).

Reply:
you can add a new group in a follow on PR